scrapli_community/huawei/smartax/huawei_smartax.py

```diff
@@ -29,19 +29,30 @@
             pattern=r"^\S{1,48}#$",
             name="privilege_exec",
             previous_priv="exec",
-            deescalate="quit",
+            deescalate="disable",
```

Comment:
pretty much same question as above on this -- will "disable" pretty much always work or are there times disable would be an unrecognized command? if the latter we could create a variant for newer things maybe.

Reply:
copy, works for me!